cloud/pkg/cloudhub/servers/server.go

```diff
@@ -37,13 +37,13 @@ func StartCloudHub(config *util.Config, messageq *channelq.ChannelMessageQueue)
 	handler.InitHandler(config, messageq)
 
 	svc := server.Server{
-		Type:       api.ProtocolTypeQuic,
+		Type:       protocolType,
 		Addr:       fmt.Sprintf("%s:%d", config.Address, config.QuicPort),
 		TLSConfig:  &tlsConfig,
 		AutoRoute:  true,
 		ConnNotify: handler.CloudhubHandler.OnRegister,
 		ExOpts:     api.QuicServerOption{MaxIncomingStreams: config.MaxIncomingStreams},
```

Comment:
for websocket, it is `api.WSServerOption{Path: "/"},`

Reply:
done